config/routes.rb

```diff
@@ -1,22 +1,14 @@
 Webapp::Application.routes.draw do
 
-  resources :home do
+  resources :assignments, only: [] do
     collection do
-      get :welcome
-    end
-  end
-
-  resources :assignments do
-    collection do
-      get :index
       get :knight
     end
   end
 
-  resources :regions do
+  resources :regions, except: [:show] do
     get 'recipients', :on => :member
     get 'request_rescue', :on => :member
-    as_routes
```

Comment:
It loos like `as_routes` comes from ActiveScaffold. Did you make sure that's not messed up at all? I can't really tell how to check that 🙃